### DISCUSSION TITLE
Measure grpc response size

### DISCUSSION THREAD

**Discussion by manastare-kasada** [bojand/ghz#443] [Q&A] [answered]
Hi Team, 

I am using ghz to measure the size of the payload returned from a unary GRPC service. I cannot find documentation/examples on how to do this. I have tried using `runner.WithStreamRecvMsgIntercept()` without any success. Any help would be appreciated.

**Comment by bojand** [comment, marked as answer]
Hello, `WithStreamRecvMsgIntercept()` works only with streaming requests, and not unary calls. There is no currently a way to add an interceptor to unary request. I agree it would be useful; in addition to tracking request size (see #4). I'll try and add this at some point in the near future.

**Comment by manastare-kasada** [comment]
Thanks for the quick reply @bojand
